<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecorderWidget</class>
 <widget class="QWidget" name="RecorderWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>252</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <spacer name="horizontalSpacerLeft">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <widget class="QFrame" name="frameClock">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>451</width>
       <height>120</height>
      </size>
     </property>
     <widget class="QLCDNumber" name="lcdNumber">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>10</y>
        <width>430</width>
        <height>100</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>100</height>
       </size>
      </property>
      <property name="numDigits">
       <number>11</number>
      </property>
      <property name="digitCount">
       <number>11</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
      <property name="intValue" stdset="0">
       <number>888888888</number>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumberShadowMinute">
      <property name="geometry">
       <rect>
        <x>126</x>
        <y>18</y>
        <width>84</width>
        <height>84</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="numDigits">
       <number>2</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
      <property name="value" stdset="0">
       <double>88.000000000000000</double>
      </property>
      <property name="intValue" stdset="0">
       <number>88</number>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumberShadowHour">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>18</y>
        <width>84</width>
        <height>84</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="numDigits">
       <number>2</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
      <property name="value" stdset="0">
       <double>88.000000000000000</double>
      </property>
      <property name="intValue" stdset="0">
       <number>88</number>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumberShadowSecond">
      <property name="geometry">
       <rect>
        <x>239</x>
        <y>18</y>
        <width>84</width>
        <height>84</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="numDigits">
       <number>2</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
      <property name="value" stdset="0">
       <double>88.000000000000000</double>
      </property>
      <property name="intValue" stdset="0">
       <number>88</number>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumberShadowMillisecond">
      <property name="geometry">
       <rect>
        <x>353</x>
        <y>18</y>
        <width>84</width>
        <height>84</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="numDigits">
       <number>2</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
      <property name="value" stdset="0">
       <double>88.000000000000000</double>
      </property>
      <property name="intValue" stdset="0">
       <number>88</number>
      </property>
     </widget>
     <zorder>lcdNumberShadowMillisecond</zorder>
     <zorder>lcdNumberShadowMinute</zorder>
     <zorder>lcdNumberShadowHour</zorder>
     <zorder>lcdNumberShadowSecond</zorder>
     <zorder>lcdNumber</zorder>
    </widget>
   </item>
   <item row="1" column="3">
    <spacer name="horizontalSpacerRight">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="2">
    <layout class="QHBoxLayout" name="horizontalLayoutPlayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <spacer name="horizontalSpacerLeftPlayout">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>150</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="icon">
        <iconset resource="Resource.qrc">
         <normaloff>:/Graphics/Images/Record.png</normaloff>:/Graphics/Images/Record.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerRightPlayout">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>150</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="2">
    <layout class="QHBoxLayout" name="horizontalLayoutInput">
     <item>
      <spacer name="horizontalSpacerLeftInput">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelFilename">
       <property name="text">
        <string>Filename:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditFilename">
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="placeholderText">
        <string>output.mov</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelCodec">
       <property name="text">
        <string>Codec:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxCodec">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>H.264</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DNxHD</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ProRes</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerRightInput">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <spacer name="verticalSpacerTop">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <spacer name="verticalSpacerBottom">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>42</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>RecorderWidget</receiver>
   <slot>buttonPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>303</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>buttonPressed()</slot>
 </slots>
</ui>
